test/Jobs/PauseTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+namespace Resque\Plugins\Jobs\Tests;
+
+use Credis_Client;
+use Resque;
+use Resque\Plugins\Jobs\Pause;
+use PHPUnit_Framework_TestCase;
+
+/**
+ * Job tests.
+ *
+ * @package     Resque/Tests
+ * @author      Wedy Chainy <askwedi@wedipedia.org>
+ * @license     http://www.opensource.org/licenses/mit-license.php
+ */
+class JobTest extends PHPUnit_Framework_TestCase
+{
+    protected $resque;
+    protected $redis;
+
+    public function setUp()
+    {
+        $config = file_get_contents(REDIS_CONF);
```

Comment:
So it turns out resque really doesn't allow for very good dependency injection. Redis server is set by this function meaning that unless I have a mock redis instance sitting on a port I'm not going to be able to mock this.
```
	public static function setBackend($server, $database = 0)
	{
		self::$redisServer   = $server;
		self::$redisDatabase = $database;
		self::$redis         = null;
	}```